ui/shared/utils/constants.js

```diff
@@ -1582,3 +1582,18 @@ export const ACMG_RULE_SPECIFICATION_COMP_HET = [
     },
   ],
 ]
+
+// RNAseq sample tissue type mapping
+export const TISSUE_DISPLAY = {
+  WB: 'Whole Blood',
+  F: 'Fibroblast',
+  M: 'Muscle',
+  L: 'Lymphocyte',
+}
+
+export const RNASEQ_JUNCTION_PADDING = 200
+
+export const getSpliceId = (row) => {
```

Comment:
this is only ever used in project selectors, so I don't think it needs to be in the shared constants file